features/dd-sdk-android-session-replay/build.gradle.kts

```diff
@@ -103,6 +104,7 @@ dependencies {
     }
     testImplementation(libs.bundles.jUnit5)
     testImplementation(libs.bundles.testTools)
+    unmock(libs.robolectric)
```

Comment:
why do you need robolectric libs here ?

Reply:
I thought we would need this to do the unmock. I've removed it